bash/compile_python_packages.sh

```diff
@@ -14,6 +14,9 @@ python3 -m pip install --upgrade pip-tools wheel
 echo -e "🛠 deleting ${path_to_requirements}/requirements.txt"
 rm --force ${path_to_requirements}/requirements.txt
 
+# Set GDAL verion
+sed -i -e "s/GDAL==.*$/GDAL==$(gdal-config --version)/g" ${path_to_requirements}/requirements.in
+
```

Comment:
is the GDAL version here a result of `config.sh`?

Reply:
No, that's from the gdal binary (`gdal-bin`)

Reply:
ah and `gdal-bin` is listed in `config.sh` and installed via `setup.sh` which is called in every Dockerfile